src/components/learner-credit-management/data/utils.js

```diff
@@ -656,20 +676,18 @@ export const getAssignableCourseRuns = ({ courseRuns, subsidyExpirationDatetime,
     upgradeDeadline: dayjs.unix(courseRun.upgradeDeadline).toISOString(),
   }));
   const assignableCourseRunsFilter = ({
-    enrollBy, isActive, hasEnrollBy, isLateEnrollmentEligible,
+    enrollBy, start, isActive, hasEnrollBy, isLateEnrollmentEligible,
   }) => {
     let isEligibleForEnrollment = true;
+    // Determine eligibility based on the provided enrollBy, start, and enrollmentStart date
+    isEligibleForEnrollment = enrollByDateThreshold({ enrollBy, isLateRedemptionAllowed }) && startDateThreshold({
+      enrollmentStart: dayjs().toDate(),
```

Comment:
[inform] I have the field currently named as `enroll_start` within the `course_runs` metadata in Algolia (i.e., `enrollStart` here). Not sure if we want to align on one or the other. Do you have a preference? I kept it as `enroll_start` so its somewhat parity with the existing `enrollBy` (e.g., we don't say `enrollmentBy`).

Reply:
Synced asynchrounously about this, but for posterity, we are falling back to the fields you set in your [enterprise catalog PR](https://github.com/openedx/enterprise-catalog/pull/949) of `enrollStart` and `hasEnrollStart`